calculate_daily_summary.py

```diff
@@ -90,62 +101,73 @@
     ORDER BY 1;
 """
 
+Q_MD_USERS_EXPIRE = """
+    DELETE FROM daily_multi_device_users{suffix}
+    WHERE day < '{day_expiry}'::DATE;
+"""
+
 Q_GET_FIRST_UNPROCESSED_DAY = """
     SELECT (MAX(day) + '1 day'::INTERVAL) AS timestamp
-    FROM daily_multi_device_users;
+    FROM daily_multi_device_users{suffix};
 """
 
 Q_GET_FIRST_AVAILABLE_DAY = """
     SELECT MIN(timestamp)::DATE
-    FROM activity_events;
+    FROM activity_events{suffix};
 """
 
 Q_GET_LAST_AVAILABLE_DAY = """
     SELECT MAX(timestamp)::DATE
-    FROM activity_events;
+    FROM activity_events{suffix};
 """
 
 Q_VACUUM_TABLES = """
     END;
-    VACUUM FULL daily_activity_per_device;
-    VACUUM FULL daily_multi_device_users;
+    VACUUM FULL daily_activity_per_device{suffix};
+    VACUUM FULL daily_multi_device_users{suffix};
 """
 
-def summarize_events(day_from=None, day_until=None):
+def summarize_events():
     db = postgres.Postgres(DB)
     db.run("BEGIN TRANSACTION")
     try:
-        db.run(Q_DAILY_DEVICES_CREATE_TABLE)
-        db.run(Q_MD_USERS_CREATE_TABLE)
-        # By default, summarize the latest days that are not yet summarized.
-        if day_from is None:
-            day_from = db.one(Q_GET_FIRST_UNPROCESSED_DAY)
+        for suffix in TABLE_SUFFIXES:
+            db.run(Q_DAILY_DEVICES_CREATE_TABLE.format(suffix=suffix))
+            db.run(Q_MD_USERS_CREATE_TABLE.format(suffix=suffix))
+            day_expiry = db.one(Q_GET_FIRST_AVAILABLE_DAY.format(suffix=suffix))
```

Comment:
nit: it took me a while to work out how `day_expiry` worked here.  IIUC, since it's the first day of data available in the main events table, we delete any items older than that from the summary table.  I wonder if we can rename it something a little clearer like `day_earliest` or `day_$BIKESHEDCOLOR`, but not a big deal.